plugins/wallabag.koplugin/main.lua

```diff
@@ -1003,7 +1016,11 @@ end
 
 function Wallabag:onAddWallabagArticle(article_url)
     if not NetworkMgr:isOnline() then
-        NetworkMgr:promptWifiOn()
+        self:addToDownloadQueue(article_url)
+        UIManager:show(InfoMessage:new{
+            text = _("Article added to download queue"),
```

Comment:
We should keep it uniform.
```suggestion
            text = T(_("Article added to download queue:\n%1"), BD.url(article_url)),
```